query/src/test/java/io/keen/client/java/TestKeenQueryClientBuilder.java

```diff
@@ -0,0 +1,24 @@
+package io.keen.client.java;
+
+/**
+ * Created by claireyoung on 5/26/15.
+ */
+public class TestKeenQueryClientBuilder extends KeenQueryClient.QueryBuilder {
+
+    private boolean isNetworkConnected = true;
+
+    public TestKeenQueryClientBuilder(KeenProject project) {
+        super(project);
+    }
+
+    @Override
+    protected KeenJsonHandler getDefaultJsonHandler() {
+        return new TestJsonHandler();
+    }
+
+    @Override
+    protected KeenQueryClient buildInstance() {
```

Comment:
Nit: Hmm, this seems to be exactly the same as the parent implementation; why is it here?